dashboard/src/store/mockStore.js

```diff
@@ -0,0 +1,30 @@
+// import configureStore to create a mock store where we will dispatch our actions
+import configureStore from "redux-mock-store";
+// import thunk middle to make our action asynchronous
+import thunk from "redux-thunk";
+
+// initialise middlewares
+const middlewares = [thunk];
+
+// initialise MockStore which is only the configureStore method which take middlewares as its parameters
+const mockStore = configureStore(middlewares);
+
+export const mockState = {
```

Comment:
I suspect this name should be more specific, like `mockToastState`, since I expect there will be more "store" mocks in the future? I think a better name would also make the test case a little easier to read since there's less room to wonder what's really there. 😄

Reply:
The same store can be used for different reducers.